WordImpress/Give
# Conditionally display offline donation instructions based on form settings

Resolves #6019 

## Description



The offline donation instructions were still being displayed on the multi-step donation form even when the form settings were set to _disabled_. This PR fixes this issue by conditionally displaying instructions based on the form settings.

## Affects



- Renamed  `give_get_offline_payment_instruction` to  `give_get_offline_payment_instructions`
- Refactored `give_get_offline_payment_instructions` to check for the _disabled_ option
- Added helper for getting formatted instructions to `get_formatted_offline_instructions`

## Visuals



![offline-instructions](https://user-images.githubusercontent.com/10138447/136435461-42cf48b2-1e01-48fb-853b-7351a930e828.png)



## Testing Instructions



1. Globally enable Offline Donations.
2. Set it as the default.
3. Create a multi-step form, and on the individual form, select to disable offline donations.
4. Visit the form and see the offline donation instructions displayed.

## Pre-review Checklist
